packages/orchestrator-ui-components/src/components/WfoForms/formFields/ListField.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { EuiFlexItem, EuiFormRow, EuiText } from '@elastic/eui';
+import range from 'lodash/range';
+import React, { Children, cloneElement, isValidElement } from 'react';
+import { connectField, filterDOMProps, joinName, useField } from 'uniforms';
+import { ListAddField } from './ListAddField';
+
+import { listFieldStyling } from './listFieldStyling';
+import { FieldProps } from './types';
+import { ListItemField } from './ListItemField';
+
+declare module 'uniforms' {
+    interface FilterDOMProps {
+        items: never;
+        uniqueItems: never;
+        outerList: never;
+    }
+}
+filterDOMProps.register('minCount');
+filterDOMProps.register('maxCount');
+filterDOMProps.register('items');
+filterDOMProps.register('uniqueItems');
+filterDOMProps.register('outerList');
+
+export type ListFieldProps = FieldProps<
+    // eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
Can't know anything about the type here?

Reply:
At the moment not. Though i did change it to unknown for now